MiroApp.ts

```diff
@@ -66,51 +64,50 @@ export class MiroApp extends App {
 
     public getOauth2ClientInstance(): IOAuth2Client {
         if (!this.oauth2ClientInstance) {
-            this.oauth2ClientInstance = createOAuth2Client(
-                this,
-                this.oauth2Config,
-            );
+            this.oauth2ClientInstance = createOAuth2Client(this, this.oauth2Config);
         }
         return this.oauth2ClientInstance;
     }
 
     public async onEnable(): Promise<boolean> {
         this.botUsername = 'Miro-app.bot';
-        this.botUser = (await this.getAccessors()
-            .reader.getUserReader()
-            .getByUsername(this.botUsername)) as IUser;
+        this.botUser = (await this.getAccessors().reader.getUserReader().getByUsername(this.botUsername)) as IUser;
         return true;
     }
 
+    public async executeActionButtonHandler(context: UIKitActionButtonInteractionContext, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify): Promise<IUIKitResponse> {
+        const handler = new ExecuteActionButtonHandler(this, read, http, modify, persistence);
+        return await handler.run(context);
+    }
+
+    public async executeBlockActionHandler(context: UIKitBlockInteractionContext, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify): Promise<IUIKitResponse> {
+        const handler = new ExecuteBlockActionHandler(this, read, http, modify, persistence);
+        return await handler.run(context);
+      }
+
     public async executeViewSubmitHandler(context: UIKitViewSubmitInteractionContext, read: IRead, http: IHttp, persistence: IPersistence, modify: IModify) {
         const handler = new ExecuteViewSubmitHandler(this, read, http, modify, persistence);
         return await handler.run(context);
       }
 
     protected async extendConfiguration(configuration: IConfigurationExtend): Promise<void> {
         await Promise.all([this.getOauth2ClientInstance().setup(configuration), configuration.slashCommands.provideSlashCommand(new MiroCommand(this))]);
+        configuration.ui.registerButton({
+            actionId: MiscEnum.CREATE_BOARD_ACTION_ID,
+            labelI18n: 'create-board-label',
+            context: UIActionButtonContext.ROOM_ACTION
+          });
     }
 
-    private async autorizationCallback(
-        token: IAuthData,
-        user: IUser,
-        read: IRead,
-        modify: IModify,
-        http: IHttp,
-        persistence: IPersistence,
-    ) {
+    private async autorizationCallback(token: IAuthData, user: IUser, read: IRead, modify: IModify, http: IHttp, persistence: IPersistence) {
```

Comment:
Please let's make linting consistent from next PR, also stick to compact (current) format.